project-koku/koku
# [COST-4966] - Removing unleash logic for savings plan cost

## Jira Ticket

[COST-4966](https://issues.redhat.com/browse/COST-4966)

## Description

This change will remove the unleash logic for enabling customers savings plan cost over unblended for OCP on AWS. 
This means all customers will use savings plan for calculations if available else fall back to unblended cost. 

## Testing

1. Checkout Branch
2. Restart Koku
3. Add OCP on AWS providers (with savings plan cost)
4. See openshift back populate sql populated using savings plan cost

Additionally if the cloud provider has no savings plan costs we fall back to unblended

## Release Notes
- [ x] proposed release note

```markdown
* [COST-4966](https://issues.redhat.com/browse/COST-4966) Enable Savings plan costs for OCP on AWS calculations
```

Note:
Do not release till after the 1st of June
